### ISSUE TITLE
Chat2 still same problem.

### ISSUE THREAD

**Issue by Lordblanca** [WWBN/AVideo#5672]
Had to reopen this I thought that the chat2 problem was resolved.
I have a screen shot form https://demo.avide0.com/live/).ForGodSooloved777 
Vertical View
  
Horizontal view 
  

And from https://lblivestream.com/live/0/admin 
Vertical view
 
Horizontal View
 

Copied the code from demo.avideo and changed the url from Avideo to mine and still the same thing.
Demo site settings (Modified with my https)
{"backgroundImage":"https://lblivestream.com/plugin/Chat2/bg.png","backgroundColor":"#e5ddd5","charLimit":255,"separateChatPerLive":true,"showOnChannel":true,"showOnLive":true,"showOnUserVideos":true,"showOnEmbedVideos":false,"showOnMobile":true,"startMinimized":false,"startMinimizedOnMobile":true,"minWidth":"380px","maxWidth":"2000px","height":"90vh","opacityDefault":90,"opacityHover":100,"noFadeoutBallons":false,"onlineSecondsTolerance":30,"requestsTimeout":3000,"showChatOnlyForLoggedUsers":false,"disableHoverEffect":false,"disableAttachments":false,"disableMoreChatRooms":false,"disableChatSound":false,"allowUsersToUseTheirOwnEmbed":false,"layout":{"type":{"0":"Floating","useStaticLayout":"Static under video","useStaticLeft":"Static Fixed on left","useStaticRight":"Static Fixed on right","inPlayerLayout":"Inside the video player"},"value":"useStaticRight"},"disableLocalChatServer":false,"openChatButtonStyle":"","positionOnLeft":false}

**Comment by DanielnetoDotCom** [comment]
Look how it looks like to me


https://user-images.githubusercontent.com/18009293/132728901-e1167062-e920-4b6f-b64f-317a81fedceb.mp4

Based on your parameters I only think you should uncheck startMinimizedOnMobile

**Comment by Lordblanca** [comment]
Yes that is on your I-phone. my apoligized what you saw was on the tablet. And that is the demo site. the other pictures I showed you is my site. https://lblivestream.com

**Comment by DanielnetoDotCom** [comment]
Thanks, now I see this is happening on a tablet only.

I am sending a fix in a few. it will require you to update the Chat plugin to version 7.6

**Comment by Lordblanca** [comment]
ok... Thank you

**Comment by Lordblanca** [comment]
I am at chat 7.6  
![chat-7 6](https://user-images.githubusercontent.com/37937968/132770194-60a890db-5ce8-4c1f-a3aa-a1d6d12059bf.png)

**Comment by DanielnetoDotCom** [comment]
and did you see any difference?

**Comment by Lordblanca** [comment]
Yes There is a difference. After testing on different devices. Samsung Galaxy Tablet S4, Galaxy Tab A. This is best ran at 1024x768 or in Desktop Site Mode. I will send you a video of what I found. The site has some very slick features. The single tap hiding chat bar... The program does a lot of good stuff. I just want to make sure all of the basics are met before I go live.. Keep up the good work. Oh and just to let you know. The demo site would not allow me to connect to live stream. 
PS Sorry I took so long to get back in contact with you
